hw/ip/sysrst_ctrl/data/sysrst_ctrl.hjson

```diff
@@ -106,6 +106,64 @@
       width:   "1"
     }
   ],
+  features: [
+    {
+      name: "SYSRST_CTRL.COMBO_DETECT"
+      desc: '''
+            sys_rst_ctrl block can be configured to detect input key combinations
+            with optional precondition
+            '''
+    }
+    {
+      name: "SYSRST_CTRL.AUTO_BLOCK_KEY_OUTPUT"
+      desc: '''
+            sysrst_ctrl block can be configured to override the output value of specific passthrough signals,
+            depending on whether certain input signals are asserted or not
+            '''
+    }
+    {
+      name: "SYSRST_CTRL.INPUT_TRIGGERED_INTERRUPT"
+      desc: '''
+            sysrst_ctrl block can be configured to assert interrupt based on a combination of input signal transitions
+            '''
+    }
+    {
+      name: "SYSRST_CTRL.ULATRA_LOW_POWER_WAKEUP"
+      desc: '''
+            sysrst_ctrl block can programmed to detect certain specific signal transitions on the
```

Comment:
nit: `can programmed` -> `can be programmed`

Reply:
Thanks @vrozic , Fixed typo.